### ISSUE TITLE
WFP blazor Support MVVM in future release? 

### ISSUE THREAD

**Issue by GreggAtCarfac** [dotnet/aspnetcore#43163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

Hello

We find it very interesting to combine Blazor pages with wpf and are testing how far we can go with this.
I have created a multi screen wpf application with Razor pages and ViewModels (MVVM) using Microsoft.Extensions.DependencyInjection.
We notice if we change a property on 1 screen, it does not update on the other screen.
Not even when we call the Onpropertychanged event.

is this support coming in a future release?

### Describe the solution you'd like

the 2nd screen should update on the propertychange.

**Comment by javiercn** [comment]
@GreggAtCarfac thanks for contacting us.

You might be looking for [MobileBlazorBindings](https://docs.microsoft.com/en-us/mobile-blazor-bindings/). If you are looking for some way of using MVVM with Blazor components, I do not think there is anything that prevents you from doing so, but we do not have plans to more directly support the pattern in the framework.

Whether it is one or the other, none of the two pieces at this point are committed.

/cc: @Eilon

**Comment by GreggAtCarfac** [comment]
@javiercn yes this is what i am doing. but it does not work multi screen. in a desktop wpf project i have 2 screens sharing the same viewmodel but if i click the counter button 3 times on screen 1 i see the counter increase from 0 to 1, then to 2 and finally 3 but on screen 2 it remains zero. if i then press the counter button on screen 2 once  it jumps to 4. so 1 am sure they share the same viewmodel. and i do call the onproperychanged event so i would expect screen 2 to update if the counter on screen 1 button is pushed.
